Release v0.1.9

robertrico opened the pull request.

Commit:
Add TypeScript defs and update output path

- Added TypeScript declaration file for module.
- Modified the output path in the flatten function.
- Included a .gitkeep file in the output directory.
- Reduced package-lock.json size by removing entries.

Commit:
0.1.9

Commit:
Update package metadata

- Downgrade version from 0.1.9 to 0.1.8
- Modify repository URL format
- Simplify test script command

Commit:
Update ignore files and package dependencies

- Added `.npmrc` to `.gitignore` and `.npmignore`
- Downgraded version from `0.1.9` to `0.1.8`
- Changed `gm` dependency source to GitHub
- Removed `cross-spawn` from dependencies and overrides

Commit:
0.1.9

thepika206 commented on index.d.ts:
```diff
@@ -0,0 +1,7 @@
+    };
+    export default flattener;
+  }
+  
```
indentation on the closing brace is off

Commit:
Code formatting and indentation fixes

- Fixed inconsistent indentation in TypeScript and JavaScript files.
- Adjusted spacing around function declarations and calls.
- Ensured consistent use of spaces within object literals.

robertrico commented:
@prscheduler 2025-01-15T21:00-08:00

thepika206 reviewed (commented):
I approve
